### ISSUE TITLE
GenerateContentRequest.FunctionCallingConfig is missing

### ISSUE THREAD

**Issue by techjb** [googleapis/google-cloud-dotnet#13049]
This is a preview feature
https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/function-calling#functioncallingconfig-[preview]

**Comment by jskeet** [comment]
This was added in https://github.com/googleapis/google-cloud-dotnet/pull/13035, but we haven't done a release yet. (We need to decide whether we're going to do a minor version with the breaking changes, or a new major version.)

**Comment by jskeet** [comment]
(Note that it's not in GenerateContentRequest directly - you'd set the GenerateContentRequest.ToolConfig property to a ToolConfig which in turn has a FunctionCallingConfig.)

**Comment by jskeet** [comment]
This should now be available in [Google.Cloud.AIPlatform.V1](https://www.nuget.org/packages/Google.Cloud.AIPlatform.V1) version 3.0.0

**Comment by techjb** [comment]
Yes, thank you.

**Comment by techjb** [comment]
Yes, now is in the version 3.0.0. But now, im missing support for gemini 1.5 flash, not only pro:
"Unable to submit request because the forced function calling (mode = ANY) is only supported for Gemini 1.5 Pro models."

**Comment by jskeet** [comment]
Sorry, that's not something we can help with - if the client library is making the requests accurately (which I expect it is) you'll need to check the AI Platform documentation and support for further information.
